src/nvim/api/deprecated.c

```diff
@@ -23,7 +23,7 @@ String nvim_command_output(String command, Error *err)
   FUNC_API_SINCE(1)
   FUNC_API_DEPRECATED_SINCE(7)
 {
-  return nvim_exec(command, true, err);
+  return nvim_exec(0, command, true, err);
```

Comment:
Add `channel_id` to the deprecated function as well.

Reply:
done